RULES_AND_GUIDELINES.md

```diff
@@ -0,0 +1,222 @@
+1. Vehicles 1) parked on blocks, 2) inoperative on streets (flat tires, for example), are subject to a warning that the situation be remedied within two weeks or the vehicle will be towed within a month.
+1. Violations of any of the rules # 1–4 are subject to towing at the owner's expense. Violations may be reported to the Association Manager or any member of the Board of Directors. Special requests may be directed to the Board of Directors or the Manager.
+
+[^ back to top](#)
+
+## <a name="fence-requirements"></a>Fence Requirements 
+
+SOUTH CREEK III
+
+FENCE REQUIREMENTS AND SPECIFICATIONS FOR ALL REPAIRS AND REPLACEMENTS
+
+Every fence MUST be in compliance with all of the following:
+
+1. No fence may be more than 46" tall at any location, to
+maintain the height of the original fence.
+2. Every fence must match the footprint of the original
+fence perimeter. Area of the patio cannot be expanded
+or reduced and corner posts cannot be moved.
+3. Fence posts, including corner posts must be 4”x4”
+Cedar or pressure treated wood, and must be installed
+in same location as original posts.
+4. Vertical, picket boards must be 1”x8” cedar and must
+be installed with exactly 1” spacing between pickets.
+5. Horizontal, spanner boards must be 2x4" cedar.
+6. Fences CANNOT be painted, but can be sealed,
+treated or stained using a neutral, natural tone.
+
+[^ back to top](#)
+
+## <a name="accumulation-and disposal of trash"></a>Accumulation and Disposal of Trash 
+
+Trash, Recyclables, Compostables and large or hazardous items must be properly disposed of, in a timely manner, in order to protect the public health, safety and welfare of the community. Proper disposal prevents conditions that may create fire, health or safety hazards; and does not harbor pests or impair the aesthetic appearance of the community. General storage of items not designed or intended for outdoor use could constitute a violation. Items may include but are not limited to mattresses; discarded furniture and appliances; car parts and tires; wood pallets; and piled yard or food waste. However, every property and situation is unique and may be evaluated on a case-by-case basis. (As defined by Boulder Code Enforcement)
+
+[^ back to top](#)
+
+## <a name="individual-property"></a>Individual Property
+
+Individual Property (Proposed)
+Interior space, including all drywall and finishings within each unit are under the individual control of the unit owner. However, all structural components and building features (including but not limited to studs, joists and internal construction of the building) are common property and cannot be accessed or changed by an individual or within an individual property.
```

Comment:
**cannot be accessed** isn't accurate. I can access my attic. I can put a hole in my interior dry-wall and install in-wall speakers.